### ISSUE TITLE
Add Alert Component & redux State

### ISSUE THREAD

**Issue by idurar** [idurar/mern-redux-crud-app#25]

**Comment by sweetapplepumpkinpie** [comment]
Hi @idurar Can I work on this?

**Comment by idurar** [comment]
Hi @sweetapplepumpkinpie Sure , you are welcome to contribute to idurar

**Comment by sweetapplepumpkinpie** [comment]
> Hi @sweetapplepumpkinpie Sure , you are welcome to contribute to idurar

Thanks

**Comment by sweetapplepumpkinpie** [comment]
@idurar 
I can't clone this repository with following error

![image](https://github.com/idurar/idurar-erp-crm/assets/111645620/9793c9a2-794c-4651-93bf-85579e285585)

**Comment by hrutik7** [comment]
is problem still there @sweetapplepumpkinpie 
because I cloned forked repo successfully .

**Comment by sweetapplepumpkinpie** [comment]
> is problem still there @sweetapplepumpkinpie because I cloned forked repo successfully .

No, I opened a PR to resolve the problem and @idurar merged it.

**Comment by parteekcoder** [comment]
Hi @sweetapplepumpkinpie @salahlalami ,

Can you please provide some more description about this issue?

**Comment by themarufsharifi** [comment]
Hi @salahlalami what is the status of this issue, please assign this issue to me if applicable.
